src/components/FileMessage.tsx

```diff
@@ -0,0 +1,60 @@
+import { FileMessage as ChatFileMessage } from '@sendbird/chat/message';
+import styled from 'styled-components';
+import { useGroupChannelContext } from '@sendbird/uikit-react/GroupChannel/context';
+// import { FileViewerComponent } from '@sendbird/uikit-react/ui/FileViewer';
+// import {downloadFileWithUrl, noop} from '../utils';
+// import {createPortal} from 'react-dom';
+
+const Root = styled.div`
+  width: 100%;
+`;
+
+const Image = styled.img`
+  border-radius: 16px;
+  width: 100%;
+`;
+
+type Props = {
+  message: ChatFileMessage;
+};
+
+export default function FileMessage(props: Props) {
+  const { message } = props;
+  const {
+    isScrollBottomReached,
+    scrollToBottom,
+  } = useGroupChannelContext();
+
+  // const root = document.getElementById('aichatbot-widget-window');
+
+  return (
+    <Root>
+      {/*{root &&*/}
```

Comment:
Do we want to keep this?

Reply:
yeah I prefer to keep this because I highly think we will need to add Fileviewer in the future. It will be easier to use this then

Reply:
Then let's add a brief comment about it